### PR TITLE
Add Changelog for 1.8.7

### PR DESCRIPTION
**Pull request by judemars**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add changelog for 1.8.7

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.8.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+- #1150: fix bug where volume cloning topology requirements are
+  #1232: Use errors.As so we can detect wrapped errors, and check for
+  #1227: Adding new metric pdcsi_operation_errors to fetch error ([#1244](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1244), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Bugfix for empty disk type being registered in metric for Create volume function. ([#1269](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1269), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
+- Update go version to 1.19.10 ([#1271](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1271), [@saikat-royc](https://github.com/saikat-royc))
+- Updates error message to be more user friendly when PD CSI Driver encounters an disk type UNSUPPORTED_OPERATION on ControllerPublishVolume ([#1223](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1223), [@pwschuurman](https://github.com/pwschuurman))
```
Comment (judemars) [inline CHANGELOG/CHANGELOG-1.8.md:13]: I'm not sure why this is here since it says it's in 1.8.6 below. Will look into it.

**Comment by saikat-royc** [comment]
needs a changelog for https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1284

File: CHANGELOG/CHANGELOG-1.8.md (hunk 1 above)
Comment (judemars) [reply CHANGELOG/CHANGELOG-1.8.md:13]: Removed

**Pull request merged**